python/tvm/relay/op/reduce.py

```diff
@@ -376,6 +408,39 @@ def std(data, axis=None, keepdims=False, exclude=False):
     return sqrt(_make._variance(data, m, axis, keepdims, exclude))
 
 
+def unbiased_std(data, axis=None, keepdims=False, exclude=False):
+    """Computes the unbiased standard deviation of data over given axes.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        The input data
+
+    axis : None or int or tuple of int
+        Axis or axes along which a standard deviation operation is performed.
+        The default, axis=None, will compute the standard deviation of all elements in the
+        input array. If axis is negative it counts from the last to the first axis.
+
+    keepdims : bool
+        If this is set to True, the axes which are reduced are left in the result as dimensions
+        with size one.
+        With this option, the result will broadcast correctly against the input array.
+
+    exclude : bool
+        If `exclude` is true, reduction will be performed on the axes that are
+        NOT in axis instead.
+
+    Returns
+    -------
+    result : relay.Expr
+        The computed result.
+    """
+    axis = [axis] if isinstance(axis, int) else axis
+    m = mean(data, axis, True, exclude)
+
+    return sqrt(_make._unbiased_variance(data, m, axis, keepdims, exclude))
```

Comment:
I'm not very familiar with the specifics here, but, could this be replaced by the example below, to avoid repetition?
```
return sqrt(unbiased_variance(data, axis, keepdims, exclude))
```

Reply:
This replacement is of no problem. But the current implementation of `unbiased_variance` and `unbiased_std` follows how `variance` and `std` were implemented. I think taking the same implementation style will be better.

Reply:
Understood. Thanks